components/typography/metadata/typography-international-deprecated.yml

```diff
@@ -1,12 +1,12 @@
-name: Typography (Int., Deprecated I)
+name: Typography (Internationalized, Deprecated)
```

Comment:
I thought having this short helps to not have it over-flow in the sidebar as that is the title of the link to this page.

Reply:
They are not present in the sidebar. Please view the docs site here https://git.corp.adobe.com/pages/lawdavis/spectrum-css-typography/docs/

Reply:
right, got removed from the sidebar. It's fine.